app/models/brand-assets.ts

```diff
@@ -0,0 +1,12 @@
+import DS from 'ember-data';
+import OsfModel from './osf-model';
+
+const { attr } = DS;
+
+export default class BrandAssetsModel extends OsfModel {
+    @attr('string') primaryColor!: string;
+    @attr('string') secondaryColor!: string;
+    @attr('string') navbarLogoImage!: string;
+    @attr('string') heroLogoImage!: string;
+    @attr('string') heroBackgroundImage!: string;
+}
```

Comment:
you'll need to add the new model to the ember-data model registry ([more info on what/why in the ember-cli-typescript docs](https://ember-cli-typescript.com/docs/ts-guide/using-ts-effectively#string-keyed-lookups)), can copy how we do in [our other models](https://github.com/robustdj/ember-osf-web/blob/e6d4295e424feb494ecb9920211d1554d011b3e2/app/models/registration.ts#L137-L141)